_includes/index-doc-item.html

```diff
@@ -0,0 +1,17 @@
+{% assign is_external_guide = include.url | startswith: 'http' %}
+{% if is_external_guide %}
+  {% assign guide_url = include.url %}
+{% elsif content.docversion == 'latest' %}
+  {% assign guide_url = site.baseurl | append: include.url %}
+{% else %}
+  {% assign guide_url = site.baseurl | append: '/version/' | append: include.docversion | append: include.url %}
+{% endif %}
+<div class="{% if include.origin == 'quarkiverse-hub' %}quarkiverse {% endif %}grid__item docs-card {{ include.class }}">
+  <h4><a href="{{ guide_url }}" {% if is_external_guide %}target="_blank" {% endif %}>{{ include.title }}</a></h4>
+  <div class="description">{{ include.summary | markdownify }}</div>
+  <div class="keywords">{{ include.keywords }}</div>
+  <div class="categories">{{ include.categories }}</div>
+  {% if include.origin == 'quarkiverse-hub' -%}
+  <div class="origin">Quarkiverse Hub</div>
```

Comment:
is there a reason this is calling out Quarkiverse Hub?

Reply:
because the old code did.

Reply:
if/when we want to style additional origins, we can remove the condition on including the div.

Reply:
ha - it hadn't noticed we do that  as it shows up much further down. 
its just that I don't grok why its relevant to mark it with quarkiverse hub on docs ... If anything I wish we had docs per platform releases instead of having what looks like mix of in/outside platform docs.

Reply:
we could remove the quarkiverse docs at this point... we have Holly's site now..

Reply:
I agrée we should clean this up somehow but just removing quarkiverse hub docs does not feel like the right one as the platform has docs that are from quarkiverse. 

We should figure out what the docs site is trying to be? Docs for the core or platform release? Right now we have docs from parts of core and platform but not all and all config guide is only core.